tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -191,6 +191,10 @@ public:
       std::unique_ptr<bool[]> fMaskAvail; ///< Masks invalid values in the array
       std::size_t fNValidValues = 0;      ///< The sum of non-zero elements in the fMask
       RClusterIndex fFirstIndex;          ///< Index of the first value of the array
+      /// Reading arrays of complex values may require additional memory, for instance for the elements of
+      /// arrays of vectors. A pointer to the fAuxData array is passed to the field's BulkRead method.
+      /// The RBulk class does not modify the array in-between calls to the field's BulkRead method.
+      std::vector<unsigned char> fAuxData;
```

Comment:
I'm not totally happy with the name here :monocle_face:.  What about `fPrivateData` or `fPayloadData` (or similar)?
(if we decide to change it, let's also update the comments)

Reply:
I agree that the name is a bit vague. I'd slightly prefer it over `fPrivateData` or `fPayloadData`, which I think have connotations which don't fit here. I'm happy to change the name for a better one though.

Reply:
IMHO `fInternalStorage` and `fAuxData` are roughly equivalent: in both cases I would have to go and check the docstring :)

Reply:
Then let's keep the shorter one.